### ISSUE TITLE
iOS/tvOS Issues - Importing PSP content & Settings to fix speed related to 120hz displays

### ISSUE THREAD

**Issue by notmayo** [libretro/RetroArch#16570]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

Importing PSP content:
Whenever importing PSP content, only .zip files are picked up and not .iso files. Additionally, .zip files give a failure to extract error and will not run but .iso files launched manually run fine.

Speed relating to 120hz displays:
Games will run at twice to speed. The fix is to enable Audio sync & "Sync to exact content framerate" aka g-sync/free sync. Should both of these settings be enabled by default going forward?

### Expected behavior

PSP importing:
Pickup all of the supported file types

Speed:
Run games at full-speed

### Actual behavior

[What is actually happening]

### Steps to reproduce the bug

PSP importing
1. Go to Import content
2. Import your PSP content
3. Notice only .zip files import
4. Launch .zip file with the right core (ppsspp)
5. Failure to extract error message

### Bisect Results

Release

### Version/Commit

1.18(.1)

- RetroArch: [version/commit]

1.18(.1)

### Environment information

- OS: iOS & tvOS
- Compiler: C I'm assuming

**Comment by notmayo** [comment]
I have both the 1st gen AppleTV 4k and the current gen AppleTV 4k and I can test anything on these or iOS that you guys may need.

**Comment by hizzlekizzle** [comment]
I believe sync to exact content framerate is already enabled by default but if you already have a config file (i.e., from already installing RetroArch), it won't change the existing setting value.

**Comment by notmayo** [comment]
You are correct on that value now being default and the fact that it did not overwrite the file/setting.

**Comment by notmayo** [comment]
I just tested the 18.2 updated and yes PSP importing is fixed as reported.

**Comment by hizzlekizzle** [comment]
Is that with the manual scan or regular scan? if regular, they don't scan anywhere else, either.

**Comment by notmayo** [comment]
Regular, I just tested the manual scan and it imported the missing files. So this is broken upstream in the base project and not just in the appstore releases?

**Comment by zoltanvb** [comment]
Some content import was broken for all builds, but that one is fixed. I checked a PS1 CHD scan (not on iOS, but the scanning is pretty generic for all versions) and it seems to work now. Can you mention the CHD content that is not scanning? M3U won't be recognized.
